python/dolma/cli/tagger.py

```diff
@@ -91,6 +91,10 @@ class TaggerConfig:
         default=False,
         help="If true, only print the configuration and exit without running the taggers.",
     )
+    document_dir: Optional[str] = field(
```

Comment:
should we parametrize attributes too?

Reply:
We could for the sake of symmetry but it has little utility that I can see